karate-demo/pom.xml

```diff
@@ -20,6 +20,12 @@
     <!-- for a skeleton project, use the archetype: https://github.com/karatelabs/karate#quickstart !-->
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.apache.httpcomponents.client5</groupId>
+                <artifactId>httpclient5</artifactId>
+				<!-- Consistent with version from karate-core -->
+                <version>5.3</version>            
+            </dependency>  		
```

Comment:
@f-delahaye just curious, why was this needed

Reply:
`Spring-boot` comes with `httpclient5` 5.1.3 and overrides the version defined in `karate-core` (5.3)

```
$ mvn help:effective-pom -Dverbose | grep -A2 -B2 \<artifactId\>httpclient5\</artifactId\>
      <dependency>
        <groupId>org.apache.httpcomponents.client5</groupId>  <!-- org.springframework.boot:spring-boot-dependencies:2.6.6, line 953 -->
        <artifactId>httpclient5</artifactId>  <!-- org.springframework.boot:spring-boot-dependencies:2.6.6, line 954 -->
        <version>5.1.3</version>  <!-- org.springframework.boot:spring-boot-dependencies:2.6.6, line 955 -->
      </dependency>
```

Unfortunately, it looks like `karate-demo` does not start with my changes against httpclient  5.1.3 - it hangs.
Not sure why, I didn't investigate further - 

I could not get it to work with the latest spring-boot either. But forcing to version 5.3 in `karate-demo` solves the issue.

Reply:
ok thanks